pom.xml

```diff
@@ -1353,15 +1363,15 @@
     <profile>
       <id>cdh5</id>
       <activation>
+        <activeByDefault>true</activeByDefault>
         <property>
           <name>hadoop.profile</name>
           <value>cdh5</value>
         </property>
       </activation>
-      <properties>
-        <!-- test cdh5 with the same jars that were produced for default profile -->
-        <maven.main.skip>true</maven.main.skip>
```

Comment:
This has to do with how upstream Kite uses profiles. There, they always build against one set of dependencies and their profiles are only used for running unit/integration tests. That's why there weren't any code changes needed to support HDP, but doing specific builds is a lot less error prone.